config/sly1.yaml

```diff
@@ -142,6 +142,7 @@ segments:
     - [0x5aca0, asm, P2/unk_f] #? unknown file
 
     - [0x5af20, c, P2/find]
+    - [0x5b8ac, c, P2/find_padding]
```

Comment:
All good except this should be `asm`, not `c`, so Splat doesn’t make a file called find_padding.c in the source directory.

Reply:
Oops
Fixed this up and added the comment you had in the Discord as well